internal/resto/client.go

```diff
@@ -351,3 +364,55 @@ func createStopRequest(ctx context.Context, url, username, accessKey, jobID stri
 	req.SetBasicAuth(username, accessKey)
 	return req, nil
 }
+
+func createUploadAssetRequest(ctx context.Context, url, username, accessKey, jobID string, fileName string, contentType string, content []byte) (*http.Request, error) {
```

Comment:
Maybe it's better to create a structure for such a long list of input params?  Something like `type requestData struct`

Reply:
Since it's use only at one location, and is mainly to avoid having a huge function, I considered it was acceptable.
If it's a blocker for you or anyone, I can create using a struct, but the (code) overhead seems high in that case.

Reply:
If it only one place, I am ok. We can improve a bit this function `...,jobID, fileName, contentType string`. The size can be a bit less in this case.

Reply:
I've just removed the extra `string`